pkg/infra/usagestats/usage_stats.go

```diff
@@ -282,6 +297,8 @@ func (uss *UsageStatsService) sendUsageStats(ctx context.Context) error {
 	data := bytes.NewBuffer(out)
 	sendUsageStats(data)
 
+	uss.resetLiveStats()
```

Comment:
If `!uss.Cfg.ReportingEnabled` is true this code will not execute - is that expected? 

Maybe `sampleLiveStats()` should only run if `uss.Cfg.ReportingEnabled` is true?

Reply:
@marefr did a minor refactor to move this to the ticker instead